docs/query-builder.txt

```diff
@@ -869,7 +869,8 @@ Contains All Fields Example
 
 The following example shows how to use the ``all`` query
 operator with the ``where()`` query builder method to match
-documents that contain all the specified fields:
+documents that have a ``cast`` field containing all the specified
```

Comment:
Can you choose an other field name, as `casts` has as special meaning in Laravel (its a reserved word in the model). That could lead to confusion: https://laravel.com/docs/12.x/eloquent-mutators#attribute-casting

Reply:
Sure, changed to "writers"